Terminal.Gui/View/ViewScrollBar.cs

```diff
@@ -0,0 +1,635 @@
+﻿namespace Terminal.Gui;
+
+/// <summary>
+///     The scroll bar types used by this <see cref="ScrollBarView"/>.
+/// </summary>
+public enum ScrollBarType
```

Comment:
> Would this be simpler if it used [Flags]?

It's not recommended because we only use on of them and not the combination of some or all of them.

> The more I think about this, I think the right model is for ScrollBar.Enabled be the way devs turn on/off scrollbars on View.

II thought about this a lot and came to the conclusion that an enum would be more appropriate. But these are just opinions, suggestions that everyone can participate and give their opinion.

Reply:
Gotcha, however, I now think the whole idea of "View.ScrollBarType" should go away. `ScrollBar` should have a `Orientation` property (using the existing enum which is still in the `Graphs` folder). `View` should have a `public ScrollBar VerticalScrollBar { get; init; }` and a `public ScrollBar HorizontalScrollBar { get; init; }`. When these objects are created their `.Orientation` property should be set appropriately.